### PR TITLE
issue #34 Change project name to Jakarta Annotations

### PR DESCRIPTION
**Pull request by rotty3000**

### CODE REVIEW

**Comment by m0mus** [comment]
We will be able to merge it after the new name is approved.

**Comment by rotty3000** [comment]
sounds good!

**Comment by rotty3000** [comment]
@m0mus please let me know what else is immediately pending for this project and I will tackle it.

File: pom.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
     </properties>
 
     <name>${extension.name} API</name>
-    <description>Common Annotations for the JavaTM Platform API</description>
+    <description>Jakarta Annotations for the Java(TM) Platform API</description>
```
Comment (bshannon) [inline pom.xml]: It would be fine if the description was just "Jakarta Annotations".

File: pom.xml [hunk 2]
```diff
@@ -234,7 +234,7 @@
                         <Implementation-Version>${spec.implementation.version}</Implementation-Version>
                         <Specification-Version>${spec.specification.version}</Specification-Version>
                         <Bundle-Description>
-                            Java(TM) Common Annotations ${spec.version} API Design Specification
+                            Java(TM) Jakarta Annotations ${spec.version} API Design Specification
                         </Bundle-Description>
```
Comment (bshannon) [inline pom.xml]: This should just be something like "Jakarta Annotations, Version ${spec.version}".
And obviously once the specification document is available it should be changed to match.

**Pull request merged**